### PR TITLE
Brazilian translation

### PR DESCRIPTION
**Pull request by Tarkoviski**
Just translated some files, there is still a lot of work to do.

Translated:
* About
* Achievements
* Achievements/Beatmap-Packs-Achievements
* Bancho
* BanchoBot
* Beatmap-forum
* Countries-that-participated-in-osu!-tournaments
* Entry-not-found
* Mascots
* People/Language-Moderators
* Registration
* Videos-from-Youtube
* Storyboarding/Audio
* Storyboarding/Maps
* Storyboarding/Variables
* meta/Entry-not-found
* Welcome
* Adding Custom Hitsounds
* Beatmap Editor
* GL Support_Issues
* Handling Cheating
* Twitter

### CODE REVIEW

**Comment by sr229** [comment]
Is this still being worked on? Last activity is from December 2016.

**Comment by MegaApplePi** [comment]
Before I continue the review, please change all of the file's names from `pt-br.md` to `pt.md`.
(We do not use language tags.)

**Comment by Tarkoviski** [comment]
Why not?

**Comment by MegaApplePi** [comment]
We follow the [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1) language codes closely for file names. The ISO 639-1 language codes doesn't really specify the use of language tags.

It's kinda explained at <https://new.ppy.sh/wiki/Article_Style_Guide/#file-names> and <https://new.ppy.sh/wiki/Article_Style_Guide/#language-tags>.

**Comment by kj415j45** [comment]
@MegaApplePi but some problem need fix like that in Chinese
image

locale in cookies is `pt-br` but if you use `pt.md` it may return 
`Requested page could not be found for currently selected language.`
when you visit wiki

↑not sure about that↑

**Comment by MegaApplePi** [comment]
¯\_(ツ)_/¯ osu-web quirk I guess (_someone should take a closer look and patch it, if need be_)...

File: wiki/BanchoBot/pt-br.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+![Aba do BanchoBot](BanchoBot.jpg "aba do BanchoBot") **BanchoBot** é um bot (um conjunto de comandos com respostas automatizadas) programado para ajudar as pessoas na conversa exibindo certas informações e anunciando mensagens relacionadas ao jogo. O BanchoBot é o anfitrião do [IRC](IRC "wikilink") e tem um perfil chamado .
```
Comment (Kuron-kun) [inline wiki/BanchoBot/pt-br.md]: The whole file is highly outdated. Consider updating it before full review.

File: wiki/Beatmap_forum/pt-br.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        -   [Modding Queues](https://osu.ppy.sh/forum/60)
+    -   [Beatmap Graveyard](http://osu.ppy.sh/forum/19) &mdash; ***Beatmaps inativos por 4 semanas ou mais serão movidos para cá.***
+    -   [Beatmap Projects](https://osu.ppy.sh/forum/53) &mdash; ***Anuncie, discuta e participe de projetos de  beatmapping aqui.***
+
+<!-- -->
+
+-   -   [Completed Projects](https://osu.ppy.sh/forum/62) &mdash; ***Projetos concluídos podem ser arquivados e discutidos aqui.***
+-   -   [Beatmap Management](https://osu.ppy.sh/forum/115) &mdash; ***Membros do Beatmap Nomination Group discutem questões recentes aqui.***
+
+[Category:Infrastructure of osu!/pt-br](Category:Infrastructure_of_osu!/pt-br "wikilink") <Category:Website/pt-br>
```
Comment (Kuron-kun) [inline wiki/Beatmap_forum/pt-br.md]: Remove this line.

File: wiki/Beatmap_Editor/pt-br.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<img src="Compose.png" title="Editor do osu! em modo janela" alt="Editor do osu! em modo janela" width="200" />
```
Comment (Kuron-kun) [inline wiki/Beatmap_Editor/pt-br.md]: This line must have a heading saying "Editor de Beatmap", just like the english one.

File: wiki/Beatmap_Editor/pt-br.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+Artigos relacionados
+----------------
+
+-   [Beatmapping](Beatmapping "wikilink") (Noções básicas de mapeamento)
+-   [Beat Snap Divisor](Beat_Snap_Divisor "wikilink")
+-   [Distance Snap](Distance_Snap "wikilink")
+-   [Custom Sample Overrides](Custom_Sample_Overrides "wikilink")
+-   [Kiai Time](Kiai_Time "wikilink")
+
+<Category:Beatmapping/pt-br>
```
Comment (Kuron-kun) [inline wiki/Beatmap_Editor/pt-br.md]: Remove this line.

File: wiki/Beatmap_Editor/pt-br.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+-   [Song Setup](Song_Setup "wikilink"): Aqui é onde várias opções do mapa são definidas. Isso inclui cores de combo e configurações de dificuldade.
+
+Artigos relacionados
+----------------
+
+-   [Beatmapping](Beatmapping "wikilink") (Noções básicas de mapeamento)
+-   [Beat Snap Divisor](Beat_Snap_Divisor "wikilink")
+-   [Distance Snap](Distance_Snap "wikilink")
+-   [Custom Sample Overrides](Custom_Sample_Overrides "wikilink")
+-   [Kiai Time](Kiai_Time "wikilink")
```
Comment (Kuron-kun) [inline wiki/Beatmap_Editor/pt-br.md]: All these links are outdated. I'll link them below how you should add them, respectively, after each [section].

- (/wiki/Beatmaps)
- (/wiki/Beatmap_Editor/Compose)
- (/wiki/Hit_Objects)
- (/wiki/Beatmap_Editor/Design)
- (/wiki/Storyboards)
- (/wiki/Beatmap_Editor/Timing)
- (/wiki/Beatmap_Editor/Song_Setup)
- (/wiki/Beatmap_Editor/Beat_Snap_Divisor)
- (/wiki/Beatmap_Editor_Guides/Custom_Sample_Overrides)
- (/wiki/Beatmap_Editor/Distance_Snap)
- (/wiki/Beatmap_Editor/Kiai_Time)

File: wiki/Countries_that_participated_in_osu!_tournaments/pt-br.md [hunk 6]
```diff
@@ -0,0 +1,298 @@
+Este artigo lista os países que participaram nos vários torneios de osu!, Taiko, CtB e osu!mania, além de sua classificação caso tenham terminado entre as 4 melhores equipes
```
Comment (Kuron-kun) [inline wiki/Countries_that_participated_in_osu!_tournaments/pt-br.md]: Missed a dot at the end of the sentence.

File: wiki/Countries_that_participated_in_osu!_tournaments/pt-br.md [hunk 7]
```diff
@@ -0,0 +1,298 @@
...
+| Número total de equipes | 19                                          | 19                                           | 32                                           |
+
+Observações
+=====
+
+<sup>½</sup> Participou com outro país como uma única equipe.
+
+<sup>×2</sup> Tinha duas equipes.
+
+<Category:Tournaments/pt-br> [Category:History of osu!/pt-br](Category:History_of_osu!/pt-br "wikilink")
```
Comment (Kuron-kun) [inline wiki/Countries_that_participated_in_osu!_tournaments/pt-br.md]: Remove this line.

File: wiki/Handling_Cheating/pt-br.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+-   Você deu a sua conta para outra pessoa.
+-   Usou várias contas.
+-   Trapaceou (veja abaixo).
+
+O que é considerado trapaça?
+----------------------------
+
+Como as regras dizem, qualquer forma de manipulação do cliente; hack de pontuação, macro em sequencias de combo e mira automática são todas consideradas trapaça.
+
+Isso é uma regra de um golpe; trapaceiros serão **banidos permanentemente.**
```
Comment (Kuron-kun) [inline wiki/Handling_Cheating/pt-br.md]: Not sure if "regra de um golpe" is nice to read. What about "regra única" ou "regra imutável"?

File: wiki/Handling_Cheating/pt-br.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+------------------------------------------------
+
+O osu! usa métricas automatizadas combinadas com confirmação manual.
+
+A equipe nunca bane alguém incorretamente por trapacear. Antes que eles façam qualquer ação, eles se certificam de que é 100% trapaça.
+
+Meu amigo foi banido por trapacear! Ele nunca trapacearia! Vocês estão errados!
+--------------------------------------------------------------------------------------
+
+Pode ser difícil chegar a um acordo com o seu amigo, mas você terá que confiar na equipe. Peppy não está tentando estragar amizades, mas ele sabe que quase ninguém pode ser confiável. Mesmo aquelas pessoas que estão perto de você que você nunca poderia suspeitar podem agir atrás de você ou qualquer outra pessoa.
```
Comment (Kuron-kun) [inline wiki/Handling_Cheating/pt-br.md]: - "peppy" isn't capitalized.
- The last sentence uses a lot of "você". Try something like "Mesmo aquelas pessoas que estão perto de você que nunca seriam suspeitas podem agir sem que qualquer pessoa saiba."

File: wiki/Handling_Cheating/pt-br.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+Não os dê atenção indevida ou peça a outros para assistí-los.
+
+Clique no link "Report" no lado direito da pontuação do beatmap (site)
+
+Use !report na conversa do jogo para enviar uma mensagem aos moderadores.
+
+`Exemplo: !report `<Nome de usuário>` possível trapaceiro.`
+
+A sua ajuda em detectar jogo sujo é muito agradecida. Você também pode enviar um e-mail diretamente para osu@ppy.sh para uma resposta relativamente rápida.
```
Comment (Kuron-kun) [inline wiki/Handling_Cheating/pt-br.md]: jogo sujo → alguém trapaceando.

File: wiki/Mascots/pt-br.md [hunk 11]
```diff
@@ -0,0 +1,236 @@
...
+
+#### Biografia
+Don é o protagonista da série Taiko no Tatsujin. Ele é um tambor de taiko com uma borda branca e quatro patas, um rosto vermelho (que parece ser laranja) e um corpo azul claro. O sonho de Don é compartilhar a beleza do Taiko com o mundo. Três anos se passaram desde que ele se mudou para a Wada House, e ele se tornou muito popular na cidade. Ele tem um apetite monstruoso e às vezes faz compras caras na Wada House, que pode significar o pior. Ele geralmente termina as suas frases com "Ta-don", que significa "Ba-dum" em japonês.
+
+Galeria de imagens
+=============
+
+[Clique aqui para ir à galeria de imagens.](Gallery/en.md "wikilink")
+
+[Category:History of osu!/pt-br](Category:History_of_osu!/pt-br "wikilink")
```
Comment (Kuron-kun) [inline wiki/Mascots/pt-br.md]: Remove this line.

File: wiki/People/Language_Moderators/pt-br.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
+A maioria dos canais presentes no osu! têm seus próprios moderadores, que cuidam de tudo que possa invadir ou ir contra a atmosfera do canal. Embora existir vários QATs/GMTs cuidando desses problemas graças aos seus poderes, existem vários OPs da conversa que não fazem parte do QAT que moderam o canal para ajudar os outros moderadores.
```
Comment (Kuron-kun) [inline wiki/People/Language_Moderators/pt-br.md]: Before this line you forgot to add a heading "Moderadores de Idioma", just like in english.

File: wiki/Registration/pt-br.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
+Como crio uma conta?
```
Comment (Kuron-kun) [inline wiki/Registration/pt-br.md]: The whole file is highly outdated. Consider updating it before full review.

File: wiki/Storyboarding/Audio/pt-br.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
+**Arquivos de áudio** (WAV, MP3 e OGG) podem ser executados em pontos específicos do tempo. Eles são como declarações de objeto, não comandos, portanto, eles não são usados em *loops* ou *triggers*. No entanto, você pode colocá-los em qualquer uma das quatro camadas (embora não haja nenhum conceito de "sobreposição" &mdash; todos os sons serão misturados), para que você possa fazer com os sons sejam executados apenas se o jogador estiver no estado Pass ou Fail.
```
Comment (Kuron-kun) [inline wiki/Storyboarding/Audio/pt-br.md]: Same here about the heading. This one should be "Audio de Storyboard".

File: wiki/Storyboarding/Variables/pt-br.md [hunk 15]
```diff
@@ -0,0 +1,34 @@
+Você pode por sequencias de texto, até uma linha inteira, como **variáveis** para uso em outro lugar em um arquivo .osb (observe que o valor não pode mudar durante a jogabilidade, portanto, para aqueles com experiência em software, pense nisso mais como uma constante). *Arquivos .osu não suportam isso.*
```
Comment (Kuron-kun) [inline wiki/Storyboarding/Variables/pt-br.md]: Same about the heading. This one should be "Variáveis de Storyboard".

**Comment by MegaApplePi** [comment]
Just to update, @Tarkoviski, you can keep the file names as `pt-br.md`.
Article language support is now based on whatever osu-web supports.

**Pull request merged**